src/config/.yarnrc

```diff
@@ -0,0 +1,2 @@
+registry "https://registry.npmjs.org/"
+"@atlassian:registry" "https://registry.npmjs.org/"
```

Comment:
this needs to be at the top level of the project.

Reply:
moved, good catch!